platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/migration/ConfigurationMigrationService.java

```diff
@@ -36,4 +37,11 @@
      * @return MigrationWarning returned if there were non-fatal issues when exporting
      */
     Collection<MigrationWarning> export(@NotNull Path exportDirectory) throws MigrationException;
+
+    /**
+     * Get a collection of the migratables available for export
+     *
+     * @return Collection of type Describable
+     */
+    Collection<Describable> getInfo();
```

Comment:
Per discussion I'll be putting this on hold - changing implementation to only get optional migratables. Can plug-in with blueprint injected list when ready.